docs/modules/integrations/pages/github-action/upload-an-app.adoc

```diff
@@ -0,0 +1,18 @@
+= Upload an app to Kobiton
+:navtitle: Upload an app
+
+Learn how to upload an app to your app repository using GitHub Action.
+
+NOTE: For more information, see link:https://github.com/marketplace/actions/kobiton-upload-mobile-app#setup-workflow-and-action[Official GitHub Marketplace Kobiton Action].
+
+== Before you start
+
+You'll need the following:
+
+* A Kobiton account. To create a Kobiton account visit link:https://portal.kobiton.com/register[Kobiton].
+* Kobiton Username and API key. To get your Kobiton Username and API Key, follow instructions at `IV. Configure Test Script` section on link:https://kobiton.com/blog/parallel-testing-with-selenium-webdriver/[Kobiton blog].

```

Comment:
Try to link to pages within the same site. In this case it should be linked to `profile/manage-your-api-credentials#_get_an_api_key`

Reply:
Linked within the same site.